src/bin/hive_chain.rs

```diff
@@ -60,6 +60,8 @@ async fn main() -> eyre::Result<()> {
             tokio::time::sleep(tokio::time::Duration::from_millis(500)).await;
         }
 
+        // Set the block gas limit in kakarot
```

Comment:
Should this be done in this PR or later?

Reply:
I would say it can be done later if it doesn't break any hive test

Reply:
Is this a TODO then?